tests/test_common.py

```diff
@@ -648,3 +648,35 @@ def test_library(df_raw: Any, df_raw_right: Any) -> None:
         NotImplementedError, match="Cross-library comparisons aren't supported"
     ):
         df_left.join(df_right, left_on=["a"], right_on=["a"], how="inner")
+
+
+@pytest.mark.parametrize("df_raw", [df_pandas, df_polars])
```

Comment:
@MarcoGorelli I wonder if it adds any value to have the type annotation in tests if everything is `Any` 🙈

Reply:
the result of `nw.from_native` is typed, that's helped me catch some minor things